.do/deploy.template.yaml

```diff
@@ -0,0 +1,9 @@
+spec:
+  name: docker-stellar-core-horizon
+  services:
+  - dockerfile_path: Dockerfile
+    git:
+      branch: acharb-quickdeploy
```

Comment:
i'll change the branch to master after approved before deploying (in the readme button link as well)

Reply:
❓ Is it possible to not hardcode the branch so that the button works for whatever branch it is clicked from? I know Heroku support this, they use the referrer browser information to pull the branch details, but not sure if DO does. If this works the advantage of it is you can link someone to a branch, such as the CAP-21 & CAP-40 branch, and a click of the button will deploy that version rather than the master version.

Reply:
no that would be nice but DO doesn't support it like Heroku does unfortunately. For cap21/cap40 we'll have to have to change the url and `deploy.template.yaml` file to point to that cap21/cap40 branch